main.c

```diff
@@ -38,6 +38,16 @@ int main(int argc, char* argv[])
 	/*
 	 * TODO: Print if its a win or draw
 	*/
+	if(f==1)
+	printf(" Congrats X wins\n");
```

Comment:
whether i should , enclose all conditins and printf() statements in a single curly braces or , particularly enclose each printf() functions in seperate  braces??,